app/server/controllers/hearing-why.ts

```diff
@@ -6,9 +6,17 @@ import { CONST } from '../../constants';
 import { hearingWhyValidation } from '../utils/fieldValidation';
 import * as moment from 'moment';
 
+const getResponseDate = (appellantReplyDatetime?: string) => moment.utc(appellantReplyDatetime).add(6, 'weeks').format();
+
 function getIndex(req: Request, res: Response) {
-  if (req.session.newHearingConfirmationThisSession) {
-    return res.render('hearing-why/index.html', {});
+  const appellantRejected: boolean = req.session.hearing.decision && req.session.hearing.decision.appellant_reply === 'decision_rejected';
+  if (req.session.newHearingConfirmationThisSession || appellantRejected) {
```

Comment:
I think we can just set the session to rejected on successful POST of reason, and drop ```newHearingConfirmationThisSession```

Reply:
It wasn't quite that simple, but I've managed to drop `newHearingConfirmationThisSession`. Needed to allow/block access to the hearing pages based on whether a view was issued. Then set the session to reject where you said and use that date if it exists.